### PR TITLE
Created new test for Registration, added element for Signup.page.js

### PR DESCRIPTION
**Pull request by JuliaEde**

### CODE REVIEW

File: test/ui/specs/auth/registration/registerUserWithValidData.spec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const LoginPage = require('../../../pageobjects/auth/Login.page');
+const SignupPage = require('../../../pageobjects/auth/Signup.page');
+
+describe ('Registering New User With Valid Data', () => {
+
+    before('Open Login Page', async () => {
+        await LoginPage.open();
+    });
+
+    it('Should Register a New User', async () => {
```
Comment (yuliahar) [inline test/ui/specs/auth/registration/registerUserWithValidData.spec.js:10]: This spec seems to be a duplication of spec test/ui/specs/auth/signup.spec.js. Please check again.

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/ui/specs/auth/registration/registerUserWithValidData.spec.js (hunk 1 above)
Comment (JuliaEde) [reply test/ui/specs/auth/registration/registerUserWithValidData.spec.js:10]: Yes, they look very similar, but mine check registration, it is following steps from this test case https://docs.google.com/spreadsheets/d/1Eg-BVQyju1YcKWL2vJYnRgLNliGOS7Vpn982QCOd1_s/edit#gid=2&range=2:2
I was not aware that the other one exists, please let me know if I should delete it, thank you!

Comment (yuliahar) [reply test/ui/specs/auth/registration/registerUserWithValidData.spec.js:10]: Yes, this is the same TC. Please remove it and add it to the existing test/ui/specs/auth/signup.spec.js 'Activation link was sent to email.

Comment (JuliaEde) [reply test/ui/specs/auth/registration/registerUserWithValidData.spec.js]: Done, did commit and pushed!

**Review by yuliahar — APPROVED** [review]

**Pull request merged**